### PR TITLE
GasLiftGroupInfo: use a array to allow for one sum call

### PR DESCRIPTION
**Pull request by akva2**
Slightly more efficient communication is good, no?

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/wells/GasLiftGroupInfo.cpp [hunk 1]
```diff
@@ -346,10 +346,11 @@ std::tuple<double, double, double, double>
...
 {
-    double oil_rate = 0.0;
-    double water_rate = 0.0;
-    double gas_rate = 0.0;
-    double alq = 0.0;
+    std::array<double,4> rates{};
+    double& oil_rate = rates[0];
+    double& water_rate = rates[1];
+    double& gas_rate = rates[2];
+    double& alq = rates[3];
```
Comment (blattms) [inline opm/simulators/wells/GasLiftGroupInfo.cpp:353]: Use structured bindings?
```
double & [oil_rate, water_rate, gas_rate, alq] = rates;
```

Comment (atgeirr) [reply opm/simulators/wells/GasLiftGroupInfo.cpp:353]: That is looking nice, almost hard to believe it is c++!

Comment (akva2) [reply opm/simulators/wells/GasLiftGroupInfo.cpp]: i like it. structured bindings only works with auto though but i don't think that was your main point.

**Comment by akva2** [comment]
i found two more spots in wellgroupshelpers that i gave the same treatment.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks.  This is clearly an improvement&mdash;albeit possibly minor&mdash;so I'll merge into master.

**Pull request merged**